core/src/main/java/org/apache/iceberg/BaseFile.java

```diff
@@ -111,6 +112,7 @@ public PartitionData copy() {
           DataFile.SPLIT_OFFSETS,
           DataFile.EQUALITY_IDS,
           DataFile.SORT_ORDER_ID,
+          DataFile.FIRST_ROW_ID,
```

Comment:
Not a huge deal, but why add in the middle of the schema? Doesn't this just mean we have to change more accessors below?

Reply:
I kept it in the same order as the spec.